lms/static/styles/components/_LMSGrader.scss

```diff
@@ -3,11 +3,10 @@
 .LMSGrader {
   display: flex;
   flex-direction: column;
-  font-size: 16px;
+  font-size: var.$input-font-size;
```

Comment:
Good. Every time we eliminate a magic number in our SCSS source, a fairy gets their wings.